configure.ac

```diff
@@ -1845,7 +1847,7 @@ AC_MSG_RESULT([$PROFILE_TASK])
 
 llvm_bin_dir=''
 llvm_path="${PATH}"
-if test "${CC}" = "clang"
+if test "${ac_cv_cc_name}" = "clang"
```

Comment:
Consider using $CC_BASENAME here.

Reply:
CC_BASENAME may contain a platform triplet, so ac_cv_cc_name seems better.

For the same reason, when the following code does `` clang_bin=`which clang` ``, shouldn't that be `clang_bin=$CC`?

Reply:
> For the same reason, when the following code does `` clang_bin=`which clang` ``, shouldn't that be `clang_bin=$CC`?

I think I'd like to make that change in a follow-up PR.